nilearn/decomposition/tv_msdl.py

```diff
@@ -0,0 +1,114 @@
+"""
+Sparse recovery of group maps using dictionary learning
```

Comment:
Maybe something like:
'Spatial regularization of dictionary learning to obtain more interpretable group maps.'